gif-screencast.el

```diff
@@ -121,13 +121,83 @@ various programs run here."
   :init-value nil
   :keymap gif-screencast-mode-map)
 
+(defvar gif-screencast--done nil
```

Comment:
Can't we simply use the value of `git-screencast-mode` instead?

Reply:
I don't think so. Somebody can create global minor gif screencast mode in his config.

Reply:
What do you mean?  Why a global minor mode?

Reply:
I was wrong. We can use `git-screencast-mode` here.